gcp-csm-observability/src/main/java/io/grpc/gcp/csm/observability/CsmObservability.java

```diff
@@ -36,7 +36,7 @@
  * When no SDK is explicitly added no telemetry data will be collected. See
  * {@code io.opentelemetry.sdk.OpenTelemetrySdk} for information on how to construct the SDK.
  */
-@ExperimentalApi("TODO")
+@ExperimentalApi("https://github.com/grpc/grpc-java/issues/11249")
```

Comment:
Can you add the issue URL in L93 as well? Unless you are planning to create a separate issue.